src/app.js

```diff
@@ -0,0 +1,186 @@
+.option("b", {alias: "address2", describe: "Country & UK postcode - 53 char limit", type: "string", demandOption: true})
+.option("v", {alias: "vat", describe: "VAT number", type: "string", demandOption: true})
+.option("i", {alias: "image", describe: "url to image for logo", type: "string", demandOption: true})
+.option("t", {alias: "telephone", describe: "Telephone number", type: "number", demandOption: false })
+.option("w", {alias: "website", describe: "Website - 48 char limit", type: "string", demandOption: false})
+.option("t", {alias: "topMessage", describe: "Top Message - 30 char limit", type: "string", demandOption: false})
+.option("l", {alias: "locale", describe: "locale tag", type: "string", demandOption: false})
+.choices("l", ["en", "de", "zh"])
+.argv;
+ 
+// TOO: The Order info: Class that takes JSON file path - do we need to use fs function fs.createReadFileStream
+
+// check
+console.log(`Business name: ${options.name}, Address: ${options.address1 + " "  + options.address2}, VAT: ${options.vat}, Logo/Image URL: ${options.image}`);
+
+// validates mandatory CLI arguments with char limit
+const validate = (arg, limit = 53) => {
+  try {
+    if (options[arg].length > limit) {
+      throw new Error( `${arg} exceeds ${limit} character limit. Valid input required.` );
+      // throw new Error exposes an error event with two params name & message. It also terminates further execution. 
+    }
+  } catch (err) {
+      console.log(err.name, err.message);
+  }
+}
+
+validate("name", 20); 
+validate("address1");
+validate("address2");
+validate("vat");
+validate("telephone");
+validate("website");
+validate("topMessage", 30);
+
+// resize logo
+// gets original dimensions of image
+let oWidth; 
+let oHeight; 
+
+const img = new canvas.Image();
+// TODO: check if this is async
+img.onload = function() {
+  oWidth = this.width
+  oHeight = this.height;
+}
+img.src = options.image;
+
+// calculate scale  
+const logoWidth = 448;
+const logoHeight = 120;
+
+function diff (a,b) {
+  return Math.abs(a-b);
+}
+
+const determineScale = () => {
+  let widthDiff = diff(logoWidth, oWidth);
+  let heightDiff = diff(logoHeight, oHeight);
+
+  if (widthDiff <= heightDiff) {
+    return oWidth/logoWidth;  
+  } else {
+    return oHeight/logoHeight; 
+  }
+}
+let scale = determineScale(); 
+const newWidth = oWidth * scale 
+const newHeight = oHeight * scale 
+
+
+// format business info
+const formatVAT = () => {
+  return `VAT NO: ${options.vat}`;
+}
+
+const formatTelephone = () => {
+  if (options.telephone) {
+    return `Tel: ${options.telephone}`;  
+  } else {
+    return null; 
+  }
+}
+
+const formatWebsite = () => {
+  if (options.website) {
+    return `Website: ${options.website}`;  
+  } else {
+    return null; 
+  }
+}
+
+// calculate height of business info
+const marginHeight = 60;
+
+const calcBusHeight = () => {
+  let busInfoHeight = 40 + 20 + 20 + 20 + 20;  // 100 incl. line break
```

Comment:
It would help the readability of the code quite a bit if these were defined as constants, as currently they're magic numbers: https://stackoverflow.com/a/47902

It would also make it easier to edit - as it looks like at least some of the 20s are for a spacing break? If you ever want more space, then you'd only need to alter one variable.